homeassistant/components/climate/eq3btsmart.py

```diff
@@ -81,24 +102,41 @@ def set_temperature(self, **kwargs):
         self._thermostat.target_temperature = temperature
 
     @property
-    def device_state_attributes(self):
-        """Return the device specific state attributes."""
-        return {
-            ATTR_MODE: self._thermostat.mode,
-            ATTR_MODE_READABLE: self._thermostat.mode_readable,
-        }
```

Comment:
Hmm, github is showing this code here even though a newer commit re-adds device_state_attributes?! See https://github.com/home-assistant/home-assistant/pull/4959/commits/55bed24f8333354bdd7d3aa08ee112baa76b1f72 :-)